src/store/bookingPortal/index.js

```diff
@@ -85,17 +86,23 @@ const mutations = {
     'set-updated-fields'(state, fields) {
         state.updatedFields = fields;
     },
+
+    'set-booking-id'(state, id){
+        state.bookingId = id;
+    }
 };
 
 const actions = {
     async getAgents({ commit }) {
         commit('set-agent-list', await mayaClient.get('/auth/user/agents'));
     },
 
-    async getBookingDetails({ commit }, bookingId) {
+    async getBookingDetails({ commit, state }, bookingId) {
+        // Use bookingId if it's provided; otherwise, take it from state.bookingId
+        const bookingID = bookingId ? bookingId : state.bookingId
```

Comment:
how this got affected from the field mask change?

Reply:
The booking ID has been stored in the Vuex store, so if the page does not render, it will retrieve the booking ID from the store.